lamindb/models/schema.py

```diff
@@ -372,13 +367,6 @@ class Meta(Record.Meta, TracksRun.Meta, TracksUpdates.Meta):
     """A description."""
     n: int = IntegerField()
     """Number of features in the schema."""
-    itype: str | None = CharField(
-        max_length=120, db_index=True, null=True, editable=False
-    )
-    """A registry that stores feature identifiers used in this schema, e.g., `'Feature'` or `'bionty.Gene'`.
```

Comment:
Can we call out "feature identifier type"? So that people understand why it's called `itype`?